src/OpenTelemetry/.publicApi/netstandard2.0/PublicAPI.Shipped.txt

```diff
@@ -172,4 +171,3 @@ virtual OpenTelemetry.BaseProcessor<T>.OnEnd(T data) -> void
 virtual OpenTelemetry.BaseProcessor<T>.OnForceFlush(int timeoutMilliseconds) -> bool
 virtual OpenTelemetry.BaseProcessor<T>.OnShutdown(int timeoutMilliseconds) -> bool
 virtual OpenTelemetry.BaseProcessor<T>.OnStart(T data) -> void
-virtual OpenTelemetry.Logs.OpenTelemetryLoggerProvider.Dispose(bool disposing) -> void
```

Comment:
This is a bit messy - changing Dispose from virtual to override is causing the complaint from the tooling.

Reply:
As long as this is not considered an actual breaking change, we should be fine.